TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/kafka/KafkaProtobufConsumer.java

```diff
@@ -104,9 +106,17 @@ public List<TrafficStream> readNextTrafficStreamSynchronously() {
                     TrafficStream ts = TrafficStream.parseFrom(record.value());
                     // Ensure we increment trafficStreamsRead even at a higher log level
                     log.trace("Parsed traffic stream #{}: {}", trafficStreamsRead.incrementAndGet(), ts);
+                    metricsLogger.atSuccess()
+                            .addKeyValue("connectionId", ts.getConnectionId())
+                            .addKeyValue("topic-name", this.topic)
```

Comment:
Why aren't you using camel case on this one too?

Reply:
oops, no good reason. Will update.